topi/include/topi/detail/constant_utils.h

```diff
@@ -86,6 +86,26 @@ inline std::vector<int64_t> GetConstInt64Values(Array<Expr> exprs, const std::st
   return result;
 }
 
+/*!
+ * \brief Get the value of all the constant integer expressions in the given array
+ *
+ * \param values The array of integers to get the values of
+ * \param var_name The name to be used when logging an error in the event that any
+ * of the expressions are not constant integers.
+ *
+ * \return A vector of the integer values
+ */
+inline std::vector<int> GetConstIntValues(const Array<Integer>& values,
```

Comment:
Likely we don't need this, as it is already easy to get int from values, we should directly operate on Array<Integer> and avoid creating a new vector<int>